.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-13, macos-latest, windows-latest]
```

Comment:
Its kind of unfortunate that this is called `-arm` and not `-arm64` or `-aarch64` (which are the two names by which I know this architecture).  

Perhaps we can avoid directly using the OS names in artifact names?

For example, can we map `ubuntu-20.04` to `linux-x64` and `ubuntu-24.04-arm` to linux-arm64`?

Reply:
Yeah I can't say I understand the naming that GitHub went with. The only thing that might make sense sense is that since these images are "partner images" from some contract they have with ARM Ltd., it was named by a program manager? From a developer standpoint it should really be -arm64 or -aarch64. I'll see if I can come up with a not-awful mapping in the action's yml.